dev/devicelab/bin/tasks/ios_content_validation_test.dart

```diff
@@ -48,7 +48,7 @@ Future<void> main() async {
             '    • Version Number: 1.0.0\n',
             '    • Build Number: 1\n',
             '    • Display Name: Hello\n',
-            '    • Deployment Target: 11.0\n',
+            '    • Deployment Target: 12.0\n',
```

Comment:
This is the only line different than https://github.com/flutter/flutter/pull/140478